### DISCUSSION TITLE
Feature: Guided Tours on Joomla Frontend mainly used for onboarding process

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomla/joomla-cms#42014] [2. Cold Features]
Hello Super Joomlers!

Guided tours in backend is awesome! I'm not sure if it already works in Frontend. It would be nice for example an onboarding process in frontend for new customers on your joomla website or a job board using Joomla.

Just my 2 cents. Have a delightful day.

**Comment by Akshat111111** [comment]
I agree with your suggestion, You have several alternatives for implementing guided tours or onboarding procedures in the frontend of your Joomla website:
1)You may develop specialized articles or pages on your Joomla website to assist new visitors through the site's capabilities for an onboarding process. To offer a systematic onboarding experience, these articles can be linked from the homepage or menu.
2) You can also use JavaScript frameworks or modules like Shepherd or Intro.js to develop unique onboarding procedures or guided tours. Additionally, you may make a unique Joomla module to administer the tours or incorporate these scripts into the template.
